install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -819,6 +819,10 @@ data:
   dns-policy-unload-on-shutdown: {{.Values.dnsPolicyUnloadOnShutdown | quote }}
 {{- end }}
 
+{{- if hasKey .Values "tcFilterPriority" }}
+  tc-filter-priority: {{ .Values.tcFilterPriority | quote }}
+{{- end}}
```

Comment:
Please add this new `tcFilterPriority` Helm value to _install/kubernetes/cilium/values.yaml_ along with relevant documentation (and default value if applicable, or `~`). Then run `make -C install/kubernetes docs` to update _install/kubernetes/cilium/README.md_. If `tcFilterPriority` is uncommented in values.yaml, then the check should be adapted to:
```suggestion
{{- if .Values.tcFilterPriority }}
  tc-filter-priority: {{ .Values.tcFilterPriority | quote }}
{{- end}}
```

Reply:
Ok. Any recommendation for *where* in the file I put it. My thought was at the bottom. Thanks

Reply:
Good question, would it make sense to add it under `.Values.bpf`?

Reply:
Or daemon?

Reply:
I think users are more likely to look for it under `bpf.` given it's specific to BPF attachment.

Reply:
Ok. Done. Thanks.